### ISSUE TITLE
Extend length of cart connector pin pads

### ISSUE THREAD

**Issue by cristofercruz** [Staacks/gbinterceptor#14]
While soldering the cart connector to the interceptor PCB I noticed the pads only extend as far as the pins which makes it a bit harder than necessary to heat the copper pad. You might consider extending the length of those pads slightly so there is a bit of copper pad exposed at the edge where the soldering iron tip can make contact.

![pads](https://user-images.githubusercontent.com/9136454/216740473-6b4e689a-a9d1-4e63-ae53-fc8b0e49644a.jpg)

**Comment by Staacks** [comment]
Good point. I don't dare updating the PCB without testing it myself and I don't want to order a batch just for this, but I will keep it open to include when enough details have accumulated for a new PCB revision.
